light_curves/lc_classifier.ipynb

```diff
@@ -0,0 +1,983 @@
+    "nobj = df_reindex.index.unique('objectid').max() + 1\n",
+    "\n",
+    "#for each objct in the dataframe\n",
+    "for oid in tqdm(range(nobj)):\n",
+    "    try:\n",
+    "        #what is the median flux of all bands for this object?\n",
+    "        meanflux = df_reindex.loc[oid,:,:,:].flux.mean(skipna = True)\n",
+    "        stdflux = df_reindex.loc[oid,:,:,:].flux.std(skipna=True)\n",
+    "        print('mean flux', meanflux)\n",
+    "    except KeyError:\n",
+    "        #some of the data cleaning above must have removed some objects\n",
+    "        print(oid, ' objectid doesnt exist')\n",
+    "    else:\n",
+    "        querystring = f'objectid == {oid}'\n",
+    "        zflux = (df_reindex_reset.query(querystring).flux.values - meanflux) / stdflux\n",
+    "        datetime = df_reindex_reset.query(querystring).datetime.values\n",
+    "        objectid = df_reindex_reset.query(querystring).objectid.values\n",
+    "        band = df_reindex_reset.query(querystring).band.values\n",
+    "        label = df_reindex_reset.query(querystring).label.values\n",
+    "        dfsingle = pd.DataFrame(dict(flux=zflux, err=df_reindex.loc[oid,:,:,:].err, datetime=datetime, objectid=objectid, band=band,label=label))\n",
+    "\n",
+    "        #now concatenate the single new df with df_lc\n",
+    "        if oid == 0:\n",
+    "            df_norm = dfsingle\n",
+    "        else:\n",
+    "            df_norm = pd.concat([df_norm, dfsingle])\n",
+    "            \n",
+    "            \n",
+    "#reset the index\n",
+    "df_norm = df_norm.set_index([\"objectid\", \"label\", \"band\", \"datetime\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#proove this did what I wanted it to do for one randomly chosen object\n",
+    "oid = 318\n",
+    "meanflux = df_norm.loc[oid,:,:,:].flux.mean(skipna = True)\n",
+    "print(meanflux)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.5  Restructure dataframe in format expected by sktime\n",
+    "- Make columns have band names in them and then remove band from the index\n",
+    "- pivot the dataframe so that SKTIME understands its format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pivoted_df_reindex = df_reindex.reset_index(level=\"band\").pivot(columns=\"band\")\n",
+    "pivoted_df_reindex.columns = [\"_\".join(col) for col in pivoted_df_reindex.columns.values]"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "#proove this did what I wanted it to do\n",
+    "pd.set_option('display.max_rows', None)\n",
+    "display(pivoted_df_reindex.loc[4,:,:,:])\n",
+    "pd.reset_option('all')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pivoted_df_reindex"
```

Comment:
This still shows all normalized fluxes as 100 (the initialized value) for me. Not sure why they are not updated.